static/app/components/eventOrGroupHeader.tsx

```diff
@@ -140,6 +145,7 @@ const truncateStyles = css`
 
 const Title = styled('div')`
```

Comment:
Nit: might need a bit more margin right to align it with the header divider
<img width="897" alt="Screenshot 2024-10-10 at 3 39 14 PM" src="https://github.com/user-attachments/assets/09ccf092-63b2-4b60-b300-a903cede09d9">